### ISSUE TITLE
The deviceId is ESP8266

### ISSUE THREAD

**Issue by ArthurMa1978** [VSChina/iot-hub-dev-kit-client-app#2]
In the README.md, the screenshot still shows the deviceId is ESP8266

**Comment by formulahendry** [comment]
Yup, just copy-paste the overall doc from huzzah-esp8266, will replace the images as well as other things accordingly.
